docs/tutorials/02_ui/viz_layout.py

```diff
@@ -0,0 +1,57 @@
+from fury import ui
```

Comment:
You need to add a title for this example/tutorial and explain what is the goal of it

Reply:
Done with latest commit.